app/sagas/login.js

```diff
@@ -133,7 +136,7 @@ const handleLoginSuccess = function* handleLoginSuccess({ user }) {
 
 		yield RNUserDefaults.set(`${ RocketChat.TOKEN_KEY }-${ server }`, user.id);
 		yield RNUserDefaults.set(`${ RocketChat.TOKEN_KEY }-${ user.id }`, user.token);
-		yield put(setUser(user));
+		yield put(setUser({ ...user, loginEmailPassword: u.loginEmailPassword }));
```

Comment:
IMO we shouldn't keep this value on Redux, which will make these changes unnecessary